taichi/python/export_lang.cpp

```diff
@@ -139,6 +139,8 @@ void export_lang(py::module &m) {
       .def_readwrite("simplify_after_lower_access",
                      &CompileConfig::simplify_after_lower_access)
       .def_readwrite("lower_access", &CompileConfig::lower_access)
+      .def_readwrite("simplify_before_lower_access",
+                     &CompileConfig::simplify_before_lower_access)
```

Comment:
Should this be changed to `move_loop_invariant_outside_if`?

Reply:
Forget to change this haha